vscode/vscode.md

```diff
@@ -0,0 +1,108 @@
+This section outlines how to set up the application in the VS CODE IDE.
+
+## Prerequisites
+
+* Java SE 8, Java SE 11 or Java SE 17 is required.
+* Payara Server 5 is required. You can download Payara Server 5 from [here](https://www.payara.fish/downloads/).
+* The VS CODE IDE is required. You can download the VS CODE from [here](https://code.visualstudio.com/download).

```

Comment:
The correct term and capitalization is Visual Studio Code. Let's not refer to Visual Studio Code as an IDE here.